tests/net/quay_io/mod.rs

```diff
@@ -70,29 +94,13 @@ fn test_quayio_insecure() {
     assert_eq!(res, true);
 }
 
+#[cfg(feature = "test-net-private")]
 #[test]
```

Comment:
What about [`#[tokio::test]`](https://docs.rs/tokio/0.2.11/tokio/attr.test.html)? Then you wouldn't have to create a `Runtime` in each test.